portfolio.css

```diff
@@ -0,0 +1,308 @@
+html {
+	font-size: 10px;
+	font-family: 'Trebuchet MS';
+}
+
+* {
+	box-sizing: border-box;
+}
+
+h1, h2, h3, h4 {
+	margin: 0;
+}
+
+a {
+	text-decoration: none;
+	font-weight: bold;
+	text-shadow: 0 0 2px #26315f;
+	
+}
+
+a, header, .about, .portfolio, footer {
+	color: #9a4a32;
+}
+
+
+/* --- NAVBAR -------------------- */
+
+nav {
+	position: fixed;
+	top: 0;
+	width: 100%;
+	height: 60px;
+	background-color: #214aa1;
+	
+	display: flex;
+	align-items: center;
+}
+
+.nav-buttons {
+	display: flex;
+	gap: 20px;
+	
+	margin: auto 20px;
+	font-size: 2.5rem;
+}
+
+
+/* --- HEADER ---------------------- */
+
+header {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 20px;
+	align-items: center;
+	
+	background-color: #26315f;
+	height: 100%;
+	padding: 40px;	
+	margin-top: 60px;
+}
+
+.tardi-image {
```

Comment:
it is better to call this class header-image, in case you changed your mind later about what the image would be